Assignment2/CSVMakerTest.txt

```diff
@@ -0,0 +1,9 @@
+@isTest(SeeAllData=true)
+public class CSVMakerTest
+{
+    static testmethod void test()
+    {
+        CSVMaker.check('001xx5');
```

Comment:
Update these test classes for all the assignments to create test data. Yesterday the mail was not coming because the test classes will not work on real data, it will only work on test data which we will create here.